src/test/java/com/xjeffrose/xio/fixtures/JulBridge.java

```diff
@@ -0,0 +1,49 @@
+package com.xjeffrose.xio.fixtures;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.mockwebserver.Dispatcher;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import okhttp3.mockwebserver.SocketPolicy;
+import com.xjeffrose.xio.fixtures.OkHttpUnsafe;
+import com.xjeffrose.xio.application.Application;
+import com.xjeffrose.xio.bootstrap.ApplicationBootstrap;
+import com.xjeffrose.xio.pipeline.SmartHttpPipeline;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.nio.NioEventLoopGroup;
+import java.util.concurrent.TimeUnit;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Rule;
+import org.junit.rules.TestName;
+import com.xjeffrose.xio.application.ApplicationConfig;
+import io.netty.channel.ChannelHandler;
+import com.google.common.collect.ImmutableMap;
+import com.typesafe.config.ConfigFactory;
+import com.xjeffrose.xio.SSL.TlsConfig;
+import com.typesafe.config.Config;
+import okhttp3.RequestBody;
+import okhttp3.MediaType;
+import okhttp3.Protocol;
+import java.util.Arrays;
+import java.util.List;
+import com.xjeffrose.xio.client.ClientConfig;
+import org.slf4j.bridge.SLF4JBridgeHandler;
+import org.junit.BeforeClass;
+
+public abstract class JulBridge {
+  private static int dummy = -1;
+
+  public static void initialize() {
+    if (dummy == -1) {
```

Comment:
Not thread-safe; probably whole method should be synchronized, else you could initialize this twice, which I assume is bad.

I'm also not sure the JVM guarantees that `dummy` will be nonzero when this method is entered for the first time, especially if multiple threads hit it at once.

Reply:
The JLS [does guarantee initialization of nonfinal static fields](https://docs.oracle.com/javase/specs/jls/se8/html/jls-12.html#jls-12.4.2), FWIW, even if viewed from a thread not charged with loading the class.

Reply:
I'll make this `final`, and the method `synchronized`. I'm not super concerned if this runs twice as I'm just trying to shutup the `java.util.logging` messages that `MockWebServer` emits.